### PR TITLE
[Merged by Bors] - chore: adaptations for nightly-2024-06-28

### PR DESCRIPTION
**Pull request by jcommelin**

### CODE REVIEW

**Comment by semorrison** [comment]
bors merge

File: Mathlib/Combinatorics/SimpleGraph/Turan.lean [hunk 1]
```diff
@@ -124,10 +124,12 @@ lemma exists_isTuranMaximal :
 
 end Defs
 
-namespace IsTuranMaximal
+section Forward
```
Comment (jcommelin) [inline Mathlib/Combinatorics/SimpleGraph/Turan.lean:127]:
```suggestion
namespace IsTuranMaximal
```

File: Mathlib/Combinatorics/SimpleGraph/Turan.lean [hunk 2]
```diff
@@ -124,10 +124,12 @@ lemma exists_isTuranMaximal :
...
 
 variable {s t u : V} (h : G.IsTuranMaximal r)
 
+namespace IsTuranMaximal
+
```
Comment (jcommelin) [inline Mathlib/Combinatorics/SimpleGraph/Turan.lean:132]:
```suggestion
```

File: Mathlib/Combinatorics/SimpleGraph/Turan.lean [hunk 3]
```diff
@@ -296,4 +298,6 @@ theorem isTuranMaximal_iff_nonempty_iso_turanGraph (hr : 0 < r) :
     G.IsTuranMaximal r ↔ Nonempty (G ≃g turanGraph (Fintype.card V) r) :=
   ⟨fun h ↦ h.nonempty_iso_turanGraph, fun h ↦ isTuranMaximal_of_iso h.some hr⟩
 
+end Forward
+
```
Comment (jcommelin) [inline Mathlib/Combinatorics/SimpleGraph/Turan.lean:302]:
```suggestion
```

**Comment by kbuzzard** [comment]
bors merge

**Pull request closed without merging**